src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/storeModel.hpp

```diff
@@ -32,19 +32,45 @@ class StoreModel final : public AbstractHandler<std::shared_ptr<EventContext>>
      */
     std::shared_ptr<EventContext> handleRequest(std::shared_ptr<EventContext> data) override
     {
-        if (!data->cve5Buffer.data())
+        if (data->resourceType == ResourceType::CVE)
         {
-            throw std::runtime_error("CVE5 buffer is empty");
-        }
-        auto cve5Entry = cve_v5::GetEntry(data->cve5Buffer.data());
-        if (cve5Entry->cveMetadata())
-        {
-            auto cveMetadata = cve5Entry->cveMetadata();
-            if (cveMetadata->state() && cveMetadata->state()->str().compare("REJECTED") != 0)
+            if (!data->cve5Buffer.data())
+            {
+                throw std::runtime_error("CVE5 buffer is empty");
+            }
+            auto cve5Entry = cve_v5::GetEntry(data->cve5Buffer.data());
+            auto type = data->resource.contains("type") ? data->resource.at("type").get<std::string>() : "";
+            auto state = cve5Entry->cveMetadata()
+                             ? cve5Entry->cveMetadata()->state() ? cve5Entry->cveMetadata()->state()->str() : ""
+                             : "";
+
+            if ("update" == type)
+            {
+                // We clean the candidates DBs to keep the data synced
+                UpdateCVECandidates::removeVulnerabilityCandidate(cve5Entry, data->feedDatabase);
+                if ("PUBLISHED" == state)
+                {
+                    StoreRemediationsModel::updateRemediation(cve5Entry, data->feedDatabase);
+                    UpdateCVEDescription::storeVulnerabilityDescription(cve5Entry, data->feedDatabase);
+                    UpdateCVECandidates::storeVulnerabilityCandidate(cve5Entry, data->feedDatabase);
+                }
+                else if ("REJECTED" == state)
```

Comment:
Source code that utilizes if-elseif control statements must terminate with an else statement.

Reply:
Done!